js/main.js

```diff
@@ -136,11 +136,78 @@ $(document).ready(function() {
                 }
             });
         }
+	else if (n === 6) {
+            var hideShuffleText = 'chance of hiding and shuffling the above three squares '
+                'and adding 100% to every chance on the grid';
```

Comment:
should probably read 'every other chance': the chance on this square is reduced to 0% after one click, but I don't know if you want to explicitly state this